noobaa/noobaa-core
# bucket S3 access tab updated to new arch

### Explain the changes:
- bucket S3 access tab updated to a new arch

### Issues: Fixed / Gap #xxx
- 

### Testing Instructions:
Test s3 Access table:

1. Go to bucket s3 access page
2. Refresh page, loaded without errors,
3. Table display owner account
4. Go to accounts and 
5. create a new account with allowing access to all buckets: test@noobaa.com
6. Create account with allow access to following buckets unselect all test1@noobaa.com
7. Go to first.bucket s3 access tab
8. Now 2 records in table displayed: demo@noobaa.com, test@noobaa.com

Test edit S3 Access:

1. Go to bucket s3 access tab
2. Click on edit S3 Access
3. Modal displayed with two chacked disabled emails ( demo@noobaa.com, test@noobaa.com) and one unchecked enabled (test1@noobaa.com)
4. check test1@noobaa.com in the list and click cancel
5. Modal closed no changes
6. Click on edit S3 Access
7. Modal displayed with two chacked disabled emails ( demo@noobaa.com, test@noobaa.com) and one unchecked enabled (test1@noobaa.com)
8. check test1@noobaa.com in list and click save
9. Modal closed
10. Table displayed all 3 accounts
11. 

Test table Credential Details modal:

1. Click view on account  demo@noobaa.com and modal opened, Connection Details displayed: localhost, 123, abc.
2. Try copy, it works fine, click close and modal closed correctly

### Reminders:
- Create a new test - the first is the hardest
- User Experience is top priority
- Design for failure
- Keep it simple
- `npm test`
- Imagine a support call - Add diagnostics info, phonehome info, activity log events, external syslog
- Upgrade - DB schema, server platform, agents install, npm packages
